plugin/src/main/cpp/export/meta_export_plugin.cpp

```diff
@@ -475,6 +497,34 @@ String MetaEditorExportPlugin::_get_android_manifest_application_element_content
 		contents += "        <meta-data android:name=\"com.oculus.ossplash.background\" android:value=\"passthrough-contextual\" />\n";
 	}
 
+	HybridType hybrid_type = _get_hybrid_app_option_value();
+	if (hybrid_type != HYBRID_TYPE_DISABLED) {
+		contents +=
+				"        <activity android:name=\"org.godotengine.openxr.vendors.GodotPanelApp\" "
+				"android:process=\":GodotPanelApp\" "
+				"android:label=\"@string/godot_project_name_string\" "
+				"android:theme=\"@style/GodotAppSplashTheme\" "
+				"android:launchMode=\"singleInstancePerTask\" "
+				"android:excludeFromRecents=\"false\" "
+				"android:exported=\"true\" "
+				"android:screenOrientation=\"landscape\" "
+				"android:configChanges=\"orientation|keyboardHidden|screenSize|smallestScreenSize|density|keyboard|navigation|screenLayout|uiMode\" "
+				"android:resizeableActivity=\"true\" "
```

Comment:
We'll need to duplicate the logic in https://github.com/godotengine/godot/blob/master/platform/android/export/gradle_export_util.cpp#L266-L271 for the `excludeFromRecents`, `screenOrientation` and `resizeableActivity` attributes.